docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,298 @@
+
+    // Emit the event
+    binding.emit(myevent)
+        .then(response => {
+            // Treat the response
+            console.log("Event posted successfully");
+            console.log(response.data);
+        })
+        .catch(err => {
+            // Deal with errors
+            console.log("Error during event post");
+            console.error(err);
+        });
+}, 1000);
+```
+
+```dockerfile
+# File - Dockerfile
+
+FROM node:10
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+EXPOSE 8080
+CMD [ "node", "index.js" ]
+
+```
+
+Build and push the image:
+```bash
+docker build . -t path/to/image/registry/node-knative-heartbeat-source:v1
+docker push path/to/image/registry/node-knative-heartbeat-source:v1
+```
+
+Create the event display service which simply logs any cloudevents posted to it.
+```bash
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/aliok/event_display-864884f202126ec3150c5fcef437d90c@sha256:93cb4dcda8fee80a1f68662ae6bf20301471b046ede628f3c3f94f39752fbe08
+EOS
+```
+
+Create the `ContainerSource`:
+```bash
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: sources.knative.dev/v1alpha2
+kind: ContainerSource
+metadata:
+  name: test-heartbeats
+spec:
+  template:
+    spec:
+      containers:
+        - image: path/to/image/registry/node-knative-heartbeat-source:v1
+          name: heartbeats
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+EOS
+```
+
+Check the logs of the event display service. You will see a new message is pushed every second:
+```bash
+$ kubectl logs event-display-dpplv-deployment-67c9949cf9-bvjvk -c user-container
+
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: your.event.source.type
+  source: urn:event:from:your-api/resource/123
+  id: your-event-id
+  datacontenttype: application/json
+Data,
+  {
+    "hello": "World 1"
+  }
+```
+
+If you are interested in to see what is injected into the event source, check the logs of it:
+```bash
+$ kubectl logs test-heartbeats-deployment-7575c888c7-85w5t
+
+Sink URL is http://event-display.default.svc.cluster.local
+Emitting event #1
+Emitting event #2
+Event posted successfully
+Event posted successfully
+```
+
+Please note that the example code above is using _Binary_ mode for CloudEvents.
+Simply change
+```javascript
+let binding = new v1.BinaryHTTPEmitter(config);
+```
+with
+```javascript
+let binding = new v1.StructuredHTTPEmitter(config);
+```
+to employ structured mode.
+
+However, binary mode should be used in most of the cases as:
+- It is faster in terms of serialization and deserialization
+- It works better with cloudevents-aware proxies (like Knative Channels) can simply check the header instead of parsing the payload
+
+## Making use of SinkBinding
```

Comment:
I think we're recommending SinkBinding going forward, so I'd move this above ContainerSource (or maybe prefer to just document SinkBinding).

@n3wscott to verify